src/hub.html

```diff
@@ -46,6 +46,7 @@
     >
         <a-assets>
             <img id="rotation-symbol" crossorigin="anonymous" src="./assets/rotation.png">
+            <img id="button" crossorigin="anonymous" src="./assets/hud/button.9.png">
```

Comment:
The 4 images added in this PR are all non power of 2 so will get resized at runtime. The ".9" ones in particular are only 1px off in each dimension.